### ISSUE TITLE
[Bug] TensorFlow model unable to be converted to IR model

### ISSUE THREAD

**Issue by jocelyntehsr** [openvinotoolkit/openvino#17467]
- OpenVINO Source=> pip install 
- OpenVINO Version=> 2022.1
- Operating System / Platform => Win 11 64 bit
- Compiler => Cmake
- Problem classification => Model conversion from TensorFlow to IR
- Device use: => CPU
- Framework => TensorFlow
- Model name => pretrained model

##### Detailed description
I have trained a TensorFlow model with videos as its input. the input shape is [None, 20, 64, 64, 3]. The TF model has been froze and converted to a .pb file. But, when I try to convert it to IR, it shows the errors as below
```
[ ERROR ]  Exception occurred during running replacer "REPLACEMENT_ID" (<class 'openvino.tools.mo.load.tf.loader.TFLoader'>): Unexpected exception happened during extracting attributes for node sequential_9/lstm_10/strided_slice/stack.
Original exception message: index -1 is out of bounds for axis 0 with size 0
```

##### Steps to reproduce
I used command prompt and sent in this line of code: (I specified the None as 128)

`mo --input_model "C:\Users\user\Downloads\cnnlstm.pb" --input_shape [128,20,64,64,3]`

The .pb file is here: [https://drive.google.com/file/d/1A9GijRhrtBTSSCn4qBls8Suap3wwGXOv/view?usp=sharing](url)

##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [ / ] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [ ] There is reproducer code and related data files: images, videos, models, etc.

**Comment by rkazants** [comment]
Hi @jocelyntehsr, this bug is resolved in the upcoming OpenVINO release 2023.0: https://github.com/openvinotoolkit/openvino/pull/17327

You can anticipate this release or build OV on your own. Or you can downgrade your TensorFlow version to 2.9 for which this bug is not observed.

Best regards,
Roman

**Comment by jocelyntehsr** [comment]
> Hi @jocelyntehsr, this bug is resolved in the upcoming OpenVINO release 2023.0: #17327
> 
> You can anticipate this release or build OV on your own. Or you can downgrade your TensorFlow version to 2.9 for which this bug is not observed.
> 
> Best regards, Roman

Hi Roman, thanks for the reply. May I ask would the upcoming release support VPU devices? Thanks!

**Comment by rkazants** [comment]
> > Hi @jocelyntehsr, this bug is resolved in the upcoming OpenVINO release 2023.0: #17327
> > You can anticipate this release or build OV on your own. Or you can downgrade your TensorFlow version to 2.9 for which this bug is not observed.
> > Best regards, Roman
> 
> Hi Roman, thanks for the reply. May I ask would the upcoming release support VPU devices? Thanks!

@moslex, @mlyashko, please help with the question about VPU release.

Best regards,
Roman

**Comment by avitial** [comment]
For internal tracking of this bug, Ref. 107641
<br>

@jocelyntehsr according to the [2022.3 LTS release notes](https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino-2022-3-lts-relnotes.html#inpage-nav-1-1), OpenVINO Runtime will not support the listed devices in the 2023.0 release:

- Intel® Neural Compute Stick 2
- Intel® Vision Accelerator Design with Intel® Movidius™ Vision Processing Units (HDDL)
- AI Edge Computing Board with Intel® Movidius™ Myriad™ X C0 VPU, MYDX x 1

Note: Intel® Movidius ™ VPU based products are not supported in 2022.3 LTS release, but will be added back in a future OpenVINO 2022.3.1 LTS update. In the meantime, for support on those products please use OpenVINO 2022.1.
